frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -235,8 +236,22 @@ class Sidebar extends Component<Props> {
                         </div>
                     }
                     { accounts && accounts.map(this.getAccountLink) }
+                    
                     <div className="sidebarHeaderContainer end">
-                        <span className="sidebarHeader">{t('sidebar.settings')}</span>
+                        {/*<span className="sidebarHeader">{t('sidebar.settings')}</span>*/}
```

Comment:
Is that on purpose? where di settings go :)

Reply:
plan is to remove this category title, so yes it is on purpose. I hesitated to remove it completely as it will stay in the translations as a zombie entry forever, not sure how to really properly get rid of it

Reply:
ok, then remove the line altogether?

Reply:
cleanup and removed in a5ad9de29c2bcf023b59a214e859370f6bda0623